src/components/StackBlitzPreview/StackBlitzPreview.jsx

```diff
@@ -2,18 +2,19 @@ import PropTypes from 'prop-types';
 import openButton from '../../assets/open-in-stackblitz-button.svg';
 
 StackBlitzPreview.propTypes = {
+  description: PropTypes.string,
   example: PropTypes.string,
 };
 
 export default function StackBlitzPreview(props = {}) {
-  const { example = '' } = props;
+  const { example = '', description = '' } = props;
```

Comment:
```suggestion
  const { example = '', description = 'Check out this guide live on StackBlitz.' } = props;
```
I think It can default to the original one for most of the time.